WI #557: Normative Requirements Refinement - Part 2 (#798)
FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec: specification/columns/pricingunit.md

The PricingUnit column adheres to the following requirements:

* PricingUnit MUST be present in a [*FOCUS dataset*](#glossary:FOCUS-dataset).
* PricingUnit MUST be of type String.
* PricingUnit MUST conform to [StringHandling](#stringhandling) requirements.
* PricingUnit SHOULD conform to [UnitFormat](#unitformat) requirements.
* PricingUnit nullability is defined as follows:

>   * PricingUnit MUST be null when [ChargeCategory](#chargecategory) is "Tax".
>   * PricingUnit MUST NOT be null when ChargeCategory is "Usage" or "Purchase" and [ChargeClass](#chargeclass) is not "Correction".
>   * PricingUnit MAY be null in all other cases.

Author: refer to [this conversation ](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/pull/798/files#r2001081250) for reasoning

```suggestion
  * PricingUnit MUST be null when PricingQuantity is null.
  * PricingUnit MUST NOT be null when PricingQuantity is not null.
```

Author: Since this was already discussed and approved by TF-1, I’m committing the suggested change and resolving this conversation.
